freqtrade/exchange/okx.py

```diff
@@ -19,15 +22,17 @@ class Okx(Exchange):
         "ohlcv_candle_limit": 300,
         "mark_ohlcv_timeframe": "4h",
         "funding_fee_timeframe": "8h",
+        "can_fetch_multiple_tiers": False,
     }
 
     _supported_trading_mode_margin_pairs: List[Tuple[TradingMode, MarginMode]] = [
         # TradingMode.SPOT always supported and not required in this list
         # (TradingMode.MARGIN, MarginMode.CROSS),
```

Comment:
I think we should only change this once we know we can place orders on the exchange.

okex fails when creating an order (while i've fixed one error for now, it's still not really working)

Reply:
Whats the error you're receiving?

Reply:
multiple?
i've fixed one, ran into the next one ... simply put - it's clear that freqtrade never made a single live futures trade on okex - hence i don't think we're ready to uncomment this line.

While we could use this PR to fix these problems (and i did to some extend) - i don't think we should - as this should really be about "leverage" / "leverage tiers", not about "let's get okx working".

Reply:
I've fixed these errors now ... it's additional parameters that are needed for creating orders in OKX (as well as a different parameter set for `set_leverage()`.